src/main/java/org/broadinstitute/hellbender/tools/exome/germlinehmm/xhmm/XHMMEmissionProbabilityCalculator.java

```diff
@@ -10,30 +10,28 @@
 import org.broadinstitute.hellbender.utils.param.ParamUtils;
 
 import javax.annotation.Nonnull;
+import java.util.Random;
 
 /**
  * Implements the {@link TargetLikelihoodCalculator} interface for the original XHMM-based germline model.
  *
  * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
  */
 public final class XHMMEmissionProbabilityCalculator implements TargetLikelihoodCalculator<XHMMEmissionData> {
-    private final RandomGenerator rng;
     private final double emissionStandardDeviation;
     private final double deletionMean;
     private final double duplicationMean;
     private static final double NEUTRAL_MEAN = 0.0;
 
-    public XHMMEmissionProbabilityCalculator(final double deletionMean, final double duplicationMean, final double emissionStdDev,
-                                             @Nullable final RandomGenerator rng) {
+    public XHMMEmissionProbabilityCalculator(final double deletionMean, final double duplicationMean, final double emissionStdDev) {
         this.deletionMean = ParamUtils.isNegativeOrZero(deletionMean, "Deletion coverage shift must be negative.");
         this.duplicationMean = ParamUtils.isPositiveOrZero(duplicationMean, "Duplication coverage shift must be positive");
         emissionStandardDeviation = ParamUtils.isPositive(emissionStdDev, "Emission standard deviation must be positive");
-        this.rng = rng;
     }
 
     @Override
     public double logLikelihood(@Nonnull final XHMMEmissionData emissionData, final double copyRatio, final Target target) {
-        return new NormalDistribution(rng, getEmissionMean(copyRatio), emissionStandardDeviation)
+        return new NormalDistribution(getEmissionMean(copyRatio), emissionStandardDeviation)
```

Comment:
According to the documentation, passing null as the rng to NormalDistribution will avoid some overhead.